dist/cleanup/index.js

```diff
@@ -1,7 +1,7 @@
 /******/ (() => { // webpackBootstrap
 /******/ 	var __webpack_modules__ = ({
 
-/***/ 777:
```

Comment:
I don't think cleanup was modified, does it need to be re-compiled?

Reply:
Not really, but I guess it's still good so that it benefits from the latest ncc changes